e2e/manifests/values/deployment-scheduler.yaml

```diff
@@ -32,47 +32,6 @@ scheduler:
 lvmd:
   managed: false
 
-node:
-  volumeMounts:
-    topolvmNode:
-      - name: node-plugin-dir
-        mountPath: /run/topolvm
-      - name: lvmd-socket-dir
-        mountPath: /run/lvmd
-      - name: pod-volumes-dir
-        mountPath: /var/lib/kubelet/pods
-        mountPropagation: "Bidirectional"
-      - name: csi-plugin-dir
-        mountPath: /var/lib/kubelet/plugins/kubernetes.io/csi
-        mountPropagation: "Bidirectional"
-      - name: device-dir
-        mountPath: /dev
-  volumes:
-    - name: registration-dir
-      hostPath:
-        path: /var/lib/kubelet/plugins_registry/
-        type: Directory
-    - name: node-plugin-dir
-      hostPath:
-        path: /var/lib/kubelet/plugins/topolvm.cybozu.com/node
-        type: DirectoryOrCreate
-    - name: csi-plugin-dir
-      hostPath:
-        path: /var/lib/kubelet/plugins/kubernetes.io/csi
-        type: DirectoryOrCreate
-    - name: pod-volumes-dir
-      hostPath:
-        path: /var/lib/kubelet/pods/
-        type: DirectoryOrCreate
-    - name: lvmd-socket-dir
-      hostPath:
-        path: /tmp/topolvm
```

Comment:
We use `/tmp/topolvm` to store a lvmd socket, which is set by the kind configuration.
https://github.com/topolvm/topolvm/blob/1dad6f5a0389163027a0c061cf431ef3cf1412e5/e2e/topolvm-cluster.yaml#L49
In this case, lvmds running on the host use `/tmp/topolvm/lvmd[1-3]/lvmd.sock`, kind nodes mounts these paths on `/tmp/topolvm`, so topolvm-node containers need to mount them.
There are two possible solutions.
The first is to update values.yaml to use `/tmp/topolvm`.
The second is to update the kind configuration to mount the host path on `/run/topolvm` that is a default of helm charts.
I assume the second one can work and reduce values.yaml, but the first one is recommended because it avoid facing  other problems.

Reply:
My mistake! I misread the location `/tmp/topolvm` as `/run/topolovm`. Fixing.

Reply:
I've added the change, but I think it's still not right. Earlier manifest mounts `lvmd-socket-dir` at `/run/lvmd` and now it'll end up at `/run/topolvm`. Further, I'm puzzled how `topolvm-node` interacts with `/tmp/topolvm/lvmd[1-3]/lvmd.sock` since I don't see where those locations are being passed to it.

Reply:
Socket paths on the host `/tmp/topolvm/lvmd[1-3]/lvmd.sock ` is mapped to `/tmp/topolvm/lvmd.sock` for each node by these settings.
https://github.com/topolvm/topolvm/blob/d64dda32ef7f0d7bcbd5ad9d4e5ee5a0447380b1/e2e/topolvm-cluster.yaml#L49-L50
https://github.com/topolvm/topolvm/blob/d64dda32ef7f0d7bcbd5ad9d4e5ee5a0447380b1/e2e/topolvm-cluster.yaml#L58-L59
https://github.com/topolvm/topolvm/blob/d64dda32ef7f0d7bcbd5ad9d4e5ee5a0447380b1/e2e/topolvm-cluster.yaml#L67-L68
From k8s node perspective, each node has the socket on `/tmp/topolvm/lvmd.sock`.